src/code/z_ss_sram.c

```diff
@@ -32,20 +32,20 @@ void SsSram_Init(u32 addr, u8 handleType, u8 handleDomain, u8 handleLatency, u8
         __osRestoreInt(prevInt);
 
         sSsSramContext.ioMesg.hdr.pri = OS_MESG_PRI_NORMAL;
-        sSsSramContext.ioMesg.hdr.retQueue = &sSsSramContext.mesgQ;
+        sSsSramContext.ioMesg.hdr.retQueue = &sSsSramContext.msgQueue;
         sSsSramContext.ioMesg.devAddr = addr;
     }
 }
 
 void SsSram_Dma(void* dramAddr, size_t size, s32 direction) {
-    OSMesg mesg;
+    OSMesg msg;
 
-    osCreateMesgQueue(&sSsSramContext.mesgQ, &mesg, 1);
+    osCreateMesgQueue(&sSsSramContext.msgQueue, &msg, 1);
     sSsSramContext.ioMesg.dramAddr = dramAddr;
     sSsSramContext.ioMesg.size = size;
     osWritebackDCache(dramAddr, size);
     osEPiStartDma(&sSsSramContext.piHandle, &sSsSramContext.ioMesg, direction);
-    osRecvMesg(&sSsSramContext.mesgQ, &mesg, OS_MESG_BLOCK);
+    osRecvMesg(&sSsSramContext.msgQueue, &msg, OS_MESG_BLOCK);
```

Comment:
looks like a jank usage to store the message in the queue buffer (a=a here), thankfully it doesn't matter (`osContInit` also does it 👿, other similar queue usage pass rightfully NULL here)